cmd/juju/status/status.go

```diff
@@ -174,7 +174,8 @@ func (c *statusCommand) SetFlags(f *gnuflag.FlagSet) {
 
 	f.BoolVar(&c.color, "color", false, "Use ANSI color codes in tabular output")
 	f.BoolVar(&c.noColor, "no-color", false, "Disable ANSI color codes in tabular output")
-	f.BoolVar(&c.relations, "relations", false, "Show 'relations' section in tabular output")
```

Comment:
Do we need to add a note somewhere (visible to the user) that this flag is deprecated?